src/screens/SendSheet.js

```diff
@@ -476,13 +478,14 @@ export default function SendSheet(props) {
         : gasLimit;
 
     logger.log('gasLimit', gasLimitToUse);
+    const nonce = await getNextNonce();
```

Comment:
there is `nonce` defined on line 497 that is in the same scope

Reply:
I think for consistency, we can keep using the `nonce` from the result for what we add to the "new txn" details after submitting the txn